test/e2e/antreapolicy_test.go

```diff
@@ -2399,7 +2601,7 @@ func testACNPNamespaceIsolation(t *testing.T, data *TestData) {
 	builder2 = builder2.SetName("test-acnp-ns-isolation-applied-to-per-rule").
 		SetTier("baseline").
 		SetPriority(1.0)
-	//SetAppliedToGroup([]ACNPAppliedToSpec{{NSSelector: map[string]string{"ns": "x"}}})
+	// SetAppliedToGroup([]ACNPAppliedToSpec{{NSSelector: map[string]string{"ns": "x"}}})
```

Comment:
why not remove it?

Reply:
Sure. Will do it with other comments if there are any.